src/components/Button.js

```diff
@@ -1,12 +1,32 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ buttonName }) => (
-  <button type="button">{ buttonName }</button>
-);
+const Button = ({
+  buttonName,
+  className,
+  color,
+  wide,
+}) => {
+  if (wide) {
```

Comment:
**[Optional]**
you can use ternary operator for this;
example: `  <button className={wide ? 'wide' : 'not-wide'} style={{...}} type="button">{ name }</button>`